fintech-examples/fintech-ui/src/app/bank/payment/confirm/confirm.component.spec.ts

```diff
@@ -18,7 +21,9 @@ describe('ConfirmComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
-  });
+  /* TODO Hergie help me
```

Comment:
please refer to the bank.component.spec.ts where ActivatedRoute is mocked with `useValue` property